cmd/apps/openfaas_app_test.go

```diff
@@ -22,3 +23,77 @@ func Test_getValuesSuffix_aarch64(t *testing.T) {
 		t.Errorf("suffix, want: %s, got: %s", want, got)
 	}
 }
+
+func Test_mergeFlags(t *testing.T) {
+	tests := []struct {
+		name  string
+		flags []string
+		want  map[string]string
+	}{
+		{
+			"openfaas",
```

Comment:
Likewise, this scenario is really just about the dash within a value, update the title and remove anything that isn't required like enabled=true etc.